lib/package-managers/npm.js

```diff
@@ -44,16 +44,40 @@ function parseJson(result, data) {
     return json;
 }
 
+/**
+ * Check if package author changed between current and next version.
+ * @param {string} packageName Name of the package
+ * @param {string} currentVersion
+ * @param {string} latestVersion
+ * @returns {Promise<boolean>} A promise that fullfills with boolean value.
+ */
+async function packageAuthorChanged(packageName, currentVersion, latestVersion) {
+    npmConfig.fullMetadata = true;
+
+    const current = versionUtil.isWildCard(currentVersion.charAt(0)) ? currentVersion.substr(1) : currentVersion;
+    const latest = versionUtil.isWildCard(latestVersion.charAt(0)) ? latestVersion.substr(1) : latestVersion;
+
+    const result = await pacote.packument(packageName, npmConfig);
+    if (result.versions) {
+        const currentData = result.versions[current];
+        const latestData = result.versions[latest];
+        const currentAuthor = currentData && currentData.author ? currentData.author : null;
+        const latestAuthor = latestData && latestData.author ? latestData.author : null;
+        return !_.isEqual(currentAuthor, latestAuthor);
+    }
+    return false;
+}
+
 /**
  * Returns the value of one of the properties retrieved by npm view.
  * @param {string} packageName   Name of the package
  * @param {string} field         Field such as "versions" or "dist-tags.latest" are parsed from the pacote result (https://www.npmjs.com/package/pacote#packument)
  * @param {string} currentVersion
  * @returns {Promise}            Promised result
  */
-function viewOne(packageName, field, currentVersion) {
```

Comment:
Done!. I admit I got cooky as I am used to Typescript stuff as enforcing types, provide everything on their onw function and then export at the module definition, etc.

Reply:
No worries - refactoring in itself isn't a bad thing, but should be done in separate PR with appropriate reasoning; for huge refactors it might even make sense to open an issue first. Mixing feature and refactoring in one PR makes reviewing and tracing history harder.